docs/docs/configuration.md

```diff
@@ -308,4 +310,84 @@ modules:
 ```
 When the `no-source` parameter is `true`, the tool does not validate the `path` property, so it can be omitted as shown in the example above. No action is performed during the module's build step, so all parameters configuring the module build behaviour, e.g. `builder`, `timeout`, `ignore`, etc. are ignored. There is no content associated with this module in the result MTA archive. The rest of the module processing is the same as when the parameter is not provided, e.g. the entry in the generated the deployment descriptor is created based on `supported-platforms` settings. 
 
+#### Configuring a module that need to generate SBOM content
```

Comment:
Configuring a module that needs to generate SBOM content

Reply:
OK